erts/doc/src/erl_cmd.xml

```diff
@@ -567,6 +567,19 @@ $ <input>erl \
         <p>Replaces the path specified in the boot script; see
           <seefile marker="sasl:script"><c>script(4)</c></seefile>.</p>
       </item>
+      <tag><marker id="prevent_overlapping_partitions"/>
```

Comment:
Would it be possible to use kernel application configuration (e.g. `-kernel prevent_overlapping_partitions true|false`) instead of a command line switch?
Unlike sys.config, command lines are harder to read and write correctly, and more difficult to test.

Reply:
Yes. I just mimicked `-connect_all false` without thinking more about it.